packages/office-ui-fabric-react/src/components/Rating/Rating.Props.ts

```diff
@@ -53,6 +54,13 @@ export interface IRatingProps extends React.HTMLProps<HTMLElement> {
    * Optional id of label describing this instance of Rating
    */
   ariaLabelId?: string;
+
+  /**
+   * Optional setting to represent rating control as aggregate rating control.
+   * If set, rating control will be readonly rating control with partial star support
+   * Also if set, this overrides the disabled prop
+   */
+  aggregate?: boolean;
```

Comment:
Aggregate rating control means that the control is used to show rating which is aggregate of multiple rating values like when you show average rating. Since rating is aggregate of multiple values, it is readonly.
Since currently there is no concept of readonly control, we didnt add any support for that